src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.cs

```diff
@@ -319,14 +339,14 @@ public FixAllContext WithCancellationToken(CancellationToken cancellationToken)
             return new FixAllContext(State, this.ProgressTracker, cancellationToken);
         }
 
-        internal FixAllContext WithScope(FixAllScope scope)
```

Comment:
I cleaned up all the With'ers and now the type has a single `With` method with optional parameters.